CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.0.2 (28 November 2023)
```

Comment:
The release process is separate from specific contributions.

So perhaps we can rename this to `# Next` for now?